dcsazure_adls_to_adls_mask_pl/README.md

```diff
@@ -19,50 +19,71 @@ These linked services types are needed for the following steps:
 
 `Azure Data Lake Storage` (source) - Linked service associated with ADLS source data. This will be used for the
 following steps:
-* Switch On Copy Methodology And Supported Write (Switch activity)
-* dcsazure_adls_to_adls_mask_df/adlsSource (dataFlow)
-* dcsazure_adls_to_adls_copy_df/adlsSource (dataFlow)
```

Comment:
Pretty sure the README was wrong before - the switch used to be in the Databricks pipeline. Since we couldn't nest our `if` statements ("if dataflow for copy" & "if write version supported") we had to combine them into a variable and do a switch between the 4 different options (copy activity/write supported, copy activity/write unsupported, dataflow copy/write supported, dataflow copy/write unsupported).